pkg/logs/client/http/destination.go

```diff
@@ -376,15 +379,58 @@ func (d *Destination) updateRetryState(err error, isRetrying chan bool) bool {
 }
 
 func httpClientFactory(timeout time.Duration, cfg pkgconfigmodel.Reader) func() *http.Client {
+
+	transport := httputils.CreateHTTPTransport(cfg)
+
+	// Configure transport based on user setting
+	switch cfg.Get("logs_config.transport_type") {
+	case "http1":
+		transport.TLSNextProto = make(map[string]func(authority string, c *tls.Conn) http.RoundTripper)
```

Comment:
`http1` should not have to make changes to the transport returned from `CreateHTTPTransport` since it already forces `HTTP1.x`
These changes will override the `TLSClientConfig` previously set in `CreateHTTPTransport`